packages/expo-image-picker/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- Added support for the `cameraType` option for Android + Updated types to include the `cameraType` option in `ImagePickerOptions` ([#22143](https://github.com/expo/expo/pull/22143) by [@YoussefHenna](https://github.com/YoussefHenna))
```

Comment:
```suggestion
- Added ability to choose the camera-facing type. ([#22143](https://github.com/expo/expo/pull/22143) by [@YoussefHenna](https://github.com/YoussefHenna))
```
That option wasn't documented, so I think we should treat that as it wasn't added before on iOS.